### ISSUE TITLE
A11y_Visual Studio Code_Customizations and adding extensions_Link Purpose: "Learn more link", "Here link" in Extension page is not descriptive to understand

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122843]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code;#LinkPurpose;#JAWS;#ScreenReader;#WCAG2.4.4;;#Win32

### Environment Details:​
Application Name: Visual Studio Code
Application Version: 1.55.2 (user setup)
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome:87.04280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Screen Reader: JAWS
Standard Violated: WCAG2.4.4

### Note:
This issue is also observed at "Here" link

### Repro Steps:​

1. Turn on JAWS
2. Open Visual Studio Code editor
3. Navigate to the extension side view bar option using "Ctrl+Shift+X" key navigate to the extension side view bar option.
4. Tab till search and type "Jupyter" extension
5. Tab till Jupyter extension and press enter
6. Tab till feature contributions tab and press enter
7. Tab till settings dropdown and press enter
8. Tab till "learn more" links under description 
9. Verify whether "learn more" link is descriptive or not 

### Actual Result:
"Learn more link" in Extension page is not descriptive to understand

### Expected Result: ​
"Learn more link" in Extension page should be descriptive to understand. It should provide any related information about the learn more link.

### User Impact:
Screen Reader users will face difficulty while navigation if show link is not descriptive. They will not understand the purpose of the link.

### Attachment:
[A11y_Visual Studio Code_Customizations and adding extensions_Link Purpose Learn more link, Here link in Extension page is not descriptive to understand.zip](https://github.com/microsoft/vscode/files/6414300/A11y_Visual.Studio.Code_Customizations.and.adding.extensions_Link.Purpose.Learn.more.link.Here.link.in.Extension.page.is.not.descriptive.to.understand.zip)

**Comment by isidorn** [comment]
Disagree. I think this one is fine, "learn more" is prefectly descriptive. Closing as no change planned

**Comment by InduPriya1805** [comment]
GitHubTags:#VisualStudioCode-Win32-Apr2021;
